### ISSUE TITLE
[5.0 Feature request] Mouse over on items in tree should turn curser into pointer

### ISSUE THREAD

**Issue by slacto** [Orckestra/C1-CMS#100]
It is hard to tell if an item is clicked and opening - especially if the loading time is long. Maybe click could turn cursor to "progress".

**Comment by burningice2866** [comment]
+1

**Comment by DBailey635** [comment]
+1 from me too.

I wish GitHub had a voting system, like the one on CodePlex.

**Comment by wwwysocki** [comment]
Please see if I get your request right.

**Repro**:

1. Have a page or a 'previewable' data item that takes some time to load. (Maybe, insert some function on a page and have it wait for a few seconds before showing the contents.)
2. Click this page (or item) in the tree and wait for it to load in the browser preview.
3. Check the cursor while waiting.

**Expected**:

* The cursor is in waiting mode.

**Actual**:

* The cursor is unchanged (a standard pointer)

**Comment by slacto** [comment]
Yes, it is hard to tell if you missed the click. Especially since the cursor is not using the hand on mouse over.
